src/utils/types.ts

```diff
@@ -0,0 +1,25 @@
+export type StudentsType = {
+  id?: number;
+  lastname: string;
+  surname: string;
+  gender: 'male' | 'female' | 'others';
```

Comment:
Shouldn't it be 'other' instead of 'others'? Genuine question, I'm not sure about this.